src/test/java/com/eleks/academy/whoami/service/GameServiceImplTest.java

```diff
@@ -36,21 +34,22 @@ public void setMockMvc() {
 	}
 
 	@Test
-	void createGame() {
+	void createGameTest() {
 		final String player = "player";
 		SynchronousGame game = new PersistentGame(player, gameRequest.getMaxPlayers());
 
 		when(gameRepository.save(any(SynchronousGame.class))).thenReturn(game);
 
 		var createdGame = gameService.createGame(player, gameRequest);
+		NewGameRequest gameRequest1 = new NewGameRequest();
+		gameRequest1.setMaxPlayers(4);
 
-		assertNotNull(createdGame);
+		assertThat(gameService.createGame(player, gameRequest))
+				.isEqualTo(gameService.createGame("player", gameRequest1));
```

Comment:
here is the same `.ignoringFields("id").isEqualTo(GameDetails.builder()<fields to be set>.build()))`

Reply:
Resolved